Bottom: 4da96da1e1b8e854d8c8e28fc72c2bdee82df699
Top:    3e75ec67d1ff09ed80ecf4c7eb65209b09719be6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:55:47 +0100

Refresh of use-r-trees-to-find-and-remove

---

diff --git a/src/pour.c b/src/pour.c
index e5c232f..cf51bd9 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -734,7 +734,7 @@ touched_children_callback (const BoxType * b, void *cl)
   PolygonType *polygon = (PolygonType *) b;
 
   /* Do we actually intersect? */
-  if (isects (touched->np, polygon, False))
+  if (isects (touched->np, polygon, false))
     {
       /* Steal the clipped contours, the delete the polygon. */
       /* Add contour to local list to fiddle about with */
